### PR TITLE
Add plugin for support asynchronous storage

### PR DESCRIPTION
**Pull request by vparfonov**
### What does this PR do?

Add new plugin for support asynchronous storage. 
Will work with:
- [ ] https://github.com/eclipse/che/pull/16972 
- [x]  https://github.com/che-incubator/workspace-data-sync/pull/2
- [x] https://github.com/eclipse/che-theia/pull/771

### CODE REVIEW

File: v3/plugins/eclipse/che-async-pv-plugin/nightly/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+title: Che Async Storage Service Plugin
+description: Che Plug-in with service to provide abilty for backup/restore project source from async storage
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+repository: https://github.com/che-incubator/workspace-data-sync
+firstPublicationDate: "2020-05-01"
+category: Other
+spec:
+  containers:
+   - name: che-rsync
+     image: "vparfonov/workspace-data-sync:nightly"
```
Comment (ibuziuk) [inline v3/plugins/eclipse/che-async-pv-plugin/nightly/meta.yaml:16]: let's move it to https://quay.io/organization/eclipse

Comment (vparfonov) [reply v3/plugins/eclipse/che-async-pv-plugin/nightly/meta.yaml]: done

**Comment by ibuziuk** [comment]
[test]

**Comment by ibuziuk** [comment]
@ericwill could you please review. Not sure if failures of PR checks are expected cc @rhopp

**Comment by ibuziuk** [comment]
[ci-test]

File: v3/plugins/eclipse/che-async-pv-plugin/nightly/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+publisher: eclipse
+name: che-async-pv-plugin
+version: nightly
+type: Che Plugin
+displayName: Che Async Storage Service
+title: Che Async Storage Service Plugin
+description: Che Plug-in with service to provide abilty for backup/restore project source from async storage
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+repository: https://github.com/che-incubator/workspace-data-sync
+firstPublicationDate: "2020-05-01"
```
Comment (ibuziuk) [inline v3/plugins/eclipse/che-async-pv-plugin/nightly/meta.yaml:11]: @vparfonov worth changing the publication date

**Comment by ibuziuk** [comment]
[ci-test]

**Comment by benoitf** [comment]
in the description of the PR, it says that it depends on other issues but checkbox is not checked 

image

**Comment by AndrienkoAleksandr** [comment]
> in the description of the PR, it says that it depends on other issues but checkbox is not checked

 >@vparfonov LGTM I think this PR is not dependent on others and can be merged in the first place, right?

@vparfonov Sorry, Is your pr ready or dependent to other prs?

**Comment by ibuziuk** [comment]
> in the description of the PR, it says that it depends on other issues but checkbox is not checked

it is not true, I believe @vparfonov could you please update the description

**Comment by ibuziuk** [comment]
[ci-test]

**Pull request merged**